go-controller/pkg/ovn/master_test.go

```diff
@@ -313,8 +319,12 @@ var _ = Describe("Master Operations", func() {
 
 			_, err = config.InitConfig(ctx, fexec, nil)
 			Expect(err).NotTo(HaveOccurred())
-			config.Gateway.Mode = config.GatewayModeShared
 
+			mockOVNNBClient := mock.NewMockOVNClient(goovn.DBNB)
+			mockOVNSBClient := mock.NewMockOVNClient(goovn.DBSB)
+
+			populatePortAddresses(nodeName, hybMAC, hybIP, mockOVNNBClient)
+			config.Gateway.Mode = config.GatewayModeShared
```

Comment:
Is reordering this meaningful, or just an artifact of the changes?

Reply:
It's meaningful in the sense, the populatePortAddresses needs mockOVNNBClient to add the hybrid overlay mac and IP to the mock nbdb record.

Reply:
right, I'm asking why you moved setting `config.Gateway.Mode` later in the code.

Reply:
just an artifact of the changes.